stubs/redis/redis/client.pyi

```diff
@@ -473,7 +473,13 @@ class Redis(Generic[_StrType]):
     def watch(self, *names): ...
     def unlink(self, *names: _Key) -> int: ...
     def unwatch(self): ...
+    @overload
+    def blpop(self, keys: Union[_Value, Iterable[_Value]], timeout: Literal[0] = ...) -> Tuple[_StrType, _StrType]: ...
+    @overload
     def blpop(self, keys: Union[_Value, Iterable[_Value]], timeout: float = ...) -> Optional[Tuple[_StrType, _StrType]]: ...
```

Comment:
```suggestion
    def blpop(self, keys: Union[_Value, Iterable[_Value]], timeout: float) -> Optional[Tuple[_StrType, _StrType]]: ...
```
Otherwise the overloads overlap.